src/main/java/mate/academy/ConnectionUtil.java

```diff
@@ -0,0 +1,30 @@
+package mate.academy;
+
+import java.sql.Connection;
+import java.sql.DriverManager;
+import java.sql.SQLException;
+import java.util.Properties;
+import mate.academy.exception.DataProcessingException;
+
+public class ConnectionUtil {
+    private static final String DB_URL = "jdbc:mysql://localhost:3306/homework";
+    private static final Properties DB_PROPERTIES;
+
+    static {
+        DB_PROPERTIES = new Properties();
+        DB_PROPERTIES.put("user", "root");
+        DB_PROPERTIES.put("password", "jcudkcy1604!!!");

```

Comment:
I hope you don't use this password elsewhere :) But overall, it's a bad practice to push real credentials to your code. Or make some dummy one, like 12345